### PR TITLE
Implemented decode morse ruby program

### PR DESCRIPTION
**Pull request by vangelif**
In this PR, we: 
- Run a program using the command line.
- Used Ruby syntax for basic programming operations.
- Used Ruby to solve basic coding challenges.
- Run rubocop linters.
- Follow Gitflow.
- Wrote clean and professional code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
+# Vet Clinic Database
+
```
Comment (Gambit142) [inline README.md]: - [ ] The name of the project is not **_Vet Clinic Database_**. Kindly change the name to a more semantic name. 👍

File: README.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+**[Decode Morse]** is a project that we use Ruby to solve morse code messages and return a readable message. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+Ruby
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
```
Comment (Gambit142) [inline README.md]: - [ ] Kindly remove all placeholder texts like the one highlighted in every section of the README.

**Review by Gambit142 — COMMENTED** [review]
Hi @vangelif @clin2on3mun 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly check the comments under the review summary for reasons why I requested the change. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @vangelif & @clin2on3mun, 👋 👋  

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Decode Morse message implemented ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**